arduino_mkrzero/src/lib.rs

```diff
@@ -0,0 +1,130 @@
+#![no_std]
+
+extern crate atsamd21_hal as hal;
+
+#[cfg(feature = "rt")]
+extern crate cortex_m_rt;
+#[cfg(feature = "rt")]
+pub use cortex_m_rt::entry;
+
+#[cfg(feature = "panic_abort")]
+pub extern crate panic_abort;
```

Comment:
I have mixed feelings about this, but it does feel more ergnomic overall.  We should probably do the same for all of the board support crates, or potentially even in the `hal` crate in the spirit of DRY.